index.html

```diff
@@ -4853,6 +4853,67 @@ <h3>Issuer</h3>
         </p>
       </section>
 
+      <section class="informative">
+        <h4>Holder</h4>
+        <p>
+The value associated with the <code>holder</code> <a>property</a> is expected
+to identify an <a>holder</a> that is known to and trusted by the
+<a>verifier</a>.
+        </p>
+        <p>
+Relevant metadata about the <code>holder</code> <a>property</a> is expected
+to be available to the <a>verifier</a>. For example, an <a>holder</a> can
+publish information containing the verification material used to secure
+<a>verifiable presentations</a>. This metadata is relevant when
+checking the proofs on the <a>verifiable presentations</a>.
+        </p>
+        <p>
+See the <a href="https://www.w3.org/TR/vc-data-model/#subject-holder-relationships">Subject-Holder Relationships</a> and
```

Comment:
Not sure how to refer to this better... its no longer in our spec, but it was in the previous version... a better link here would be good.

Reply:
The original text was moved to the implementation guide: https://w3c.github.io/vc-imp-guide/#subject-holder-relationships